package.json

```diff
@@ -62,7 +62,7 @@
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-prettier": "^4.2.1",
     "ethereumjs-wallet": "^1.0.1",
-    "jest": "^29.0.0",
+    "jest": "^29.5.0",
```

Comment:
Is this related to the workflow improvements?

Reply:
No, I was testing to see if it could be related to the problem. I can revert it since it's not relevant to the purpose of this PR

Reply:
Done in [4ffc57d](https://github.com/MetaMask/KeyringController/pull/221/commits/4ffc57d0a78f8b93f2ef193c39d2564bc796ded8)